hhub/views.py

```diff
@@ -171,7 +174,7 @@ def search_entries(request):
             # total number of pages
             "page_total": paginator.num_pages,
             # current request page
-            "page_current": page,
+            "page_current": int(page),
```

Comment:
Why are casting to int here? Pagination is already trying to the desired page [here](https://github.com/gbdev/homebrewhub/blob/main/hhub/views.py#L54) so if something is wrong (e.g. page is a string) it already fails there (and goes to page 1)

Reply:
it will return a string otherwise, and... this is not consistent with the normal backend behavior

Reply:
What? That's not true. At [this line](https://github.com/gbdev/homebrewhub/blob/main/hhub/views.py#L54), page is used to paginate, if it's not a proper integer it would've failed there, so there's no way the code progress with page being a string.